examples/SingleParticleTest/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -44,21 +53,20 @@ namespace picongpu
          * http://www.tf.uni-kiel.de/matwis/amat/mw1_ge/kap_6/basics/m6_2_1.html
          */
         const float_64  GAS_DENSITY_NORMED= SI::GAS_DENSITY_SI*UNIT_LENGTH;
-     } //namespace SI
+    } //namespace SI
+    namespace particles
+    {
+        /** Number of electrons per particle (= macro particle weighting)
+         *  unit: none */
+        const float_X TYPICAL_NUM_PARTICLE_PER_MAKROPARTICLE = 1;
+    }
 #endif
 
-    /** Number of electrons per particle (= macro particle weighting)
-     *  unit: none */
-    const float_X NUM_EL_PER_PARTICLE = 1;
-
-    /** Number of ions per particle
-     *  unit: none */
-    const float_X NUM_ION_PER_PARTICLE = NUM_EL_PER_PARTICLE;
 
     /** Unit of mass */
-    const double UNIT_MASS = SI::ELECTRON_MASS_SI * double(NUM_EL_PER_PARTICLE);
+    const double UNIT_MASS = SI::ELECTRON_MASS_SI * double(particles::TYPICAL_NUM_PARTICLE_PER_MAKROPARTICLE);
     /** Unit of charge */
-    const double UNIT_CHARGE = -1.0 * SI::ELECTRON_CHARGE_SI * double(NUM_EL_PER_PARTICLE);
+    const double UNIT_CHARGE = -1.0 * SI::ELECTRON_CHARGE_SI * double(particles::TYPICAL_NUM_PARTICLE_PER_MAKROPARTICLE);
```

Comment:
No, but we should change this, I opened a issue #735 for this but I will do this after this pull request to avoid rebase conflicts.